### ISSUE TITLE
A full-scale app example

### ISSUE THREAD

**Issue by fskreuz** [ractivejs/ractivejs.github.io#74]
Because todo lists are too mainstream and are really very contrived examples. Need something that goes beyond 2-3 components and have a lot of data flowing through its veins. Suggestions are:

- Email client interface (architecture exercise)
- Hacker news front end (multi-component exercise)
- Live Twitter feed (data streaming exercise)

**Comment by dagnelies** [comment]
That sounds like fair examples. I could do that I guess. For the Email client interface, do you intended fake data or a mock server?

**Comment by fskreuz** [comment]
Yup, just fake data. Just wanted to demonstrate the creation of a sidebar component, a messages list component, a button bar, and a simple compose popup/modal/view, and that they can talk to each other. Something that's a step up from a 3-component todo list.

**Comment by PaulMaly** [comment]
I’m still working on fully isomorphic (and even server-side only) Ractive’s RealWorld example app: https://github.com/gothinkster/realworld/issues/162

I’ve not so much free time, but seems it’ll be completed soon and I’ll share it.

**Comment by dagnelies** [comment]
What about a straightforward CRUD example? ...or is that too simple? Perhaps with nested lists. I think this would cover the most fundamental use case.

**Comment by fskreuz** [comment]
I'm open to anything that's not a todo list. 😁

**Comment by dagnelies** [comment]
Same here. And there is already a TODO reference implementation anyway AFAIK

**Comment by PaulMaly** [comment]
@dagnelies 

Hm, for me RealWorld project almost is "straightforward CRUD example". You can check author's example here: https://demo.realworld.io/#/

As you can see It's also just a simple CRUD and I think it's already cover the most fundamental use cases.

**Comment by dagnelies** [comment]
As said in gitter, I started with something slightly different: a repository of short self-contained minimalist examples.

Here is the first one:

https://dagnelies.github.io/ractive-examples/user-list.html

...including a basic editor to tinker with the code.

**Comment by fskreuz** [comment]
@dagnelies By the way, Ractive has https://ractive.js.org/playground . You can skip the editor part and focus on the examples. It's the same thing that powers the "Ractive Playground" tab found to the right of the docs site.  The demos are exportable as URLs and embeddable on the docs site (see Getting Started section demos).

**Comment by dagnelies** [comment]
Here we go with a few more examples: 

https://dagnelies.github.io/ractive-examples/movies_1.html
https://dagnelies.github.io/ractive-examples/movies_2.html
https://dagnelies.github.io/ractive-examples/movies_3.html
https://dagnelies.github.io/ractive-examples/movies_4.html

...beyond that, the examples would become too complex to have an educational value

**Comment by fskreuz** [comment]
@dagnelies Repurposed the integrations page into a ginormous resources page. Your demos are now on it 🎉 

- https://ractive.js.org/resources/
- https://github.com/ractivejs/ractivejs.github.io/blob/dev/docs/resources.md

**Comment by fskreuz** [comment]
Leaving this issue open for more submissions.
